src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/Azure.Configuration.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>This is the Microsoft Azure Application Configuration Service client library</Description>
     <AssemblyTitle>Microsoft Azure.ApplicationModel.Configuration client library</AssemblyTitle>
-    <Version>1.0.0-preview.1</Version>
+    <Version>1.0.0-preview.2</Version>
```

Comment:
FYI - Once I get the work done for https://github.com/Azure/azure-sdk-for-net/issues/5753 you shouldn't need to bump these versions do get a daily build published for dogfooding/previewing.

Reply:
FYI @AlexGhiondea